packages/itwinui-react/src/utils/icons/StatusIconMap.tsx

```diff
@@ -2,17 +2,23 @@
  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
+import * as React from 'react';
 import { SvgInfoCircular } from './SvgInfoCircular.js';
 import { SvgStatusError } from './SvgStatusError.js';
 import { SvgStatusSuccess } from './SvgStatusSuccess.js';
 import { SvgStatusWarning } from './SvgStatusWarning.js';
-import type { CommonProps } from '../props.js';
```

Comment:
Yes and no. I initially made this change because I was getting "unused `React` import" errors. I ended up fixing the config to not raise these errors, but I left this change because `CommonProps` should not be used anyway.